pkgs/applications/radio/qsstv/default.nix

```diff
@@ -20,27 +20,12 @@ stdenv.mkDerivation rec {
   buildInputs = [ qtbase openjpeg fftw libpulseaudio alsaLib hamlib libv4l
                   fftwFloat ];
 
-  desktopItem = makeDesktopItem {
-    name = "QSSTV";
-    exec = "qsstv";
-    icon = "qsstv.png";
-    comment = "Qt-based slow-scan TV and fax";
-    desktopName = "QSSTV";
-    genericName = "qsstv";
-    categories = "Application;HamRadio;";
-  };
-
   installPhase = ''
-    # Install binary to the right location
-    make install INSTALL_ROOT=$out
-    mv $out/usr/bin $out/
-    rm -r $out/usr
+    # Install binary
```

Comment:
`make install` is already the default for `installPhase`. Instead of overriding `installPhase` you can now put the installation of the icon in `postInstall`.

Reply:
thanks